doc/guides/bluetooth/mesh-pics.rst

```diff
@@ -0,0 +1,312 @@
+TSPC_MESH_2_1	True		Node (C.1)
+TSPC_MESH_2_2	False (*)	Provisioner (C.1)
+==============	==============	=======================================
+
+
+Node Capabilites - Bearers
+==========================
+
+==============	==============	=======================================
+Parameter Name	Selected	Description
+==============	==============	=======================================
+TSPC_MESH_3_1	True		Advertising Bearer (C.1)
+TSPC_MESH_3_2	True		GATT Bearer (C.1)
+==============	==============	=======================================
+
+
+Provisioning
+============
+
+==============	==============	=======================================
+Parameter Name	Selected	Description
+==============	==============	=======================================
+TSPC_MESH_4_1	True		PB-ADV (C.1)
+TSPC_MESH_4_2	True		PB-GATT (C.2)
+TSPC_MESH_4_3	True		Device UUID (C.3)
+TSPC_MESH_4_4	True		Sending Unprovisioned Device Beacon (C.4)
+TSPC_MESH_4_5	True		Generic Provisioning Layer (C.3)
+TSPC_MESH_4_6	True		Provisioning Protocol (C.3)
+TSPC_MESH_4_7	False (*)	Provisioning: Public Key OOB (C.5)
+TSPC_MESH_4_8	True		Provisioning: Public Key No OOB (C.5)
+TSPC_MESH_4_9	True		Provisioning: Authentication Output OOB (C.6)
+TSPC_MESH_4_10	False (*)	Provisioning: Authentication Input OOB (C.6)
+TSPC_MESH_4_11	False (*)	Provisioning: Authentication Static OOB (C.6)
+TSPC_MESH_4_12	True		Provisioning: Authentication No OOB (C.3)
+==============	==============	=======================================
+
+
+Node Capabilites - Network Layer
+================================
+
+==============	==============	=======================================
+Parameter Name	Selected	Description
+==============	==============	=======================================
+TSPC_MESH_5_1	True		Transmitting and Receiving Secured Network
+				Layer Messages (M)
+TSPC_MESH_5_2	True		Relay Feature (C.1)
+TSPC_MESH_5_3	True		Network Message Cache (C.2)
+==============	==============	=======================================
+
+
+Node Capabilites - Lower Transport Layer
+========================================
+
+==============	==============	=======================================
+Parameter Name	Selected	Description
+==============	==============	=======================================
+TSPC_MESH_6_1	True		Transmitting and Receiving a Lower Transport
+				PDU (M)
+TSPC_MESH_6_2	True		Segmentation and Reassemply Behavior (M)
+TSPC_MESH_6_2	False (*)	Friend Cache (C.1)
+==============	==============	=======================================
+
+
+Node Capabilites - Upper Transport Layer
+========================================
+
+==============	==============	=======================================
+Parameter Name	Selected	Description
+==============	==============	=======================================
+TSPC_MESH_7_1	True		Transmitting a Secured Access Payload (M)
+TSPC_MESH_7_2	True		Receiving a Secured Uppoer Transport PDU (M)
+TSPC_MESH_7_3	False (*)	Friend Feature (C.1)
```

Comment:
Again, we support acting in Friend role, so this should be enabled. I suspect you'll then get a bunch of extra tests to run (don't worry - there shouldn't be too many surprises since I've ran them successfully manually when the support was implemented)

Reply:
Friend features enables lot of tests :)
```
MESH/NODE/CFG/HBP/BV-04-C
MESH/NODE/CFG/HBP/BV-05-C
MESH/NODE/CFG/HBP/BV-06-C
MESH/NODE/CFG/HBP/BV-07-C
MESH/NODE/CFG/HBS/BI-01-C
MESH/NODE/CFG/HBS/BV-01-C
MESH/NODE/CFG/HBS/BV-02-C
MESH/NODE/CFG/HBS/BV-03-C
MESH/NODE/CFG/HBS/BV-04-C
MESH/NODE/CFG/HBS/BV-05-C
MESH/NODE/CFG/HBS/BV-05-C-LT2
MESH/NODE/CFG/NPNPT/BI-01-C
MESH/NODE/CFG/MAKL/BI-01-C
MESH/NODE/CFG/MAKL/BI-02-C
MESH/NODE/CFG/MAKL/BI-03-C
MESH/NODE/CFG/MAKL/BI-04-C
MESH/NODE/CFG/MAKL/BV-01-C
MESH/NODE/CFG/MAKL/BV-02-C
MESH/NODE/CFG/MP/BI-01-C
MESH/NODE/CFG/MP/BI-02-C
MESH/NODE/CFG/MP/BI-03-C
MESH/NODE/CFG/MP/BV-01-C
MESH/NODE/CFG/MP/BV-02-C
MESH/NODE/CFG/NID/BI-01-C
MESH/NODE/CFG/NID/BI-03-C
MESH/NODE/CFG/NID/BV-01-C
MESH/NODE/CFG/NID/BV-02-C
MESH/NODE/CFG/NID/BV-03-C
MESH/NODE/CFG/NKL/BI-01-C
MESH/NODE/CFG/NKL/BI-02-C
MESH/NODE/CFG/NKL/BI-03-C
MESH/NODE/CFG/NKL/BV-01-C
MESH/NODE/CFG/NKL/BV-02-C
MESH/NODE/CFG/NKL/BV-03-C
MESH/NODE/CFG/NKL/BV-04-C
MESH/NODE/CFG/NTX/BV-01-C
MESH/NODE/CFG/RST/BV-01-C
MESH/NODE/CFG/SL/BI-01-C
MESH/NODE/CFG/SL/BI-02-C
MESH/NODE/CFG/SL/BI-03-C
MESH/NODE/CFG/SL/BI-04-C
MESH/NODE/CFG/SL/BI-05-C
MESH/NODE/CFG/SL/BI-06-C
MESH/NODE/CFG/SL/BI-07-C
MESH/NODE/CFG/SL/BI-08-C
MESH/NODE/CFG/SL/BI-09-C
MESH/NODE/CFG/SL/BI-10-C
MESH/NODE/CFG/SL/BV-01-C
MESH/NODE/CFG/SL/BV-02-C
MESH/NODE/CFG/SL/BV-03-C
MESH/NODE/CFG/SNBP/BI-01-C
MESH/NODE/CFG/SNBP/BV-01-C
MESH/NODE/FRND/FN/BV-01-C
MESH/NODE/FRND/FN/BV-02-C
MESH/NODE/FRND/FN/BV-02-C-LT2
MESH/NODE/FRND/FN/BV-02-C
MESH/NODE/FRND/FN/BV-02-C-LT2
MESH/NODE/FRND/FN/BV-03-C
MESH/NODE/FRND/FN/BV-03-C-LT2
MESH/NODE/FRND/FN/BV-04-C
MESH/NODE/FRND/FN/BV-04-C-LT2
MESH/NODE/FRND/FN/BV-05-C
MESH/NODE/FRND/FN/BV-06-C-LT2
MESH/NODE/FRND/FN/BV-06-C
MESH/NODE/FRND/FN/BV-06-C-LT2
MESH/NODE/FRND/FN/BV-07-C
MESH/NODE/FRND/FN/BV-08-C
MESH/NODE/FRND/FN/BV-08-C-LT2
MESH/NODE/FRND/FN/BV-09-C
MESH/NODE/FRND/FN/BV-09-C-LT2
MESH/NODE/FRND/FN/BV-10-C
MESH/NODE/FRND/FN/BV-10-C-LT2
MESH/NODE/FRND/FN/BV-11-C
MESH/NODE/FRND/FN/BV-11-C-LT2
MESH/NODE/FRND/FN/BV-12-C
MESH/NODE/FRND/FN/BV-12-C-LT2
MESH/NODE/FRND/FN/BV-13-C
MESH/NODE/FRND/FN/BV-14-C
MESH/NODE/FRND/FN/BV-14-C-LT2
MESH/NODE/FRND/FN/BV-15-C
MESH/NODE/FRND/FN/BV-15-C-LT2
MESH/NODE/FRND/FN/BV-16-C
MESH/NODE/FRND/FN/BV-16-C-LT2
MESH/NODE/FRND/FN/BV-17-C
MESH/NODE/FRND/FN/BV-17-C-LT2
MESH/NODE/FRND/FN/BV-18-C
MESH/NODE/FRND/FN/BV-18-C-LT2
MESH/NODE/FRND/FN/BV-19-C
MESH/NODE/FRND/FN/BV-19-C-LT2
MESH/NODE/FRND/FN/BV-20-C
MESH/NODE/FRND/FN/BV-20-C-LT2
MESH/NODE/FRND/FN/BV-21-C
MESH/NODE/FRND/FN/BV-21-C-LT2
MESH/NODE/FRND/FN/BI-01-C
MESH/NODE/FRND/FN/BI-02-C
MESH/NODE/FRND/FN/BI-03-C
MESH/NODE/FRND/LPN/BI-01-C
MESH/NODE/FRND/LPN/BI-02-C
MESH/NODE/FRND/LPN/BI-03-C
MESH/NODE/FRND/LPN/BV-01-C
MESH/NODE/FRND/LPN/BV-02-C
MESH/NODE/FRND/LPN/BV-03-C
MESH/NODE/FRND/LPN/BV-04-C
MESH/NODE/FRND/LPN/BV-05-C
MESH/NODE/FRND/LPN/BV-06-C
MESH/NODE/FRND/LPN/BV-07-C
MESH/NODE/FRND/LPN/BV-08-C
MESH/NODE/IVU/BI-01-C
MESH/NODE/IVU/BI-02-C
MESH/NODE/IVU/BI-04-C
MESH/NODE/IVU/BI-05-C
MESH/NODE/IVU/BI-06-C
MESH/NODE/IVU/BV-01-C
MESH/NODE/IVU/BV-02-C
MESH/NODE/IVU/BV-03-C
MESH/NODE/IVU/BV-04-C
MESH/NODE/IVU/BV-05-C
MESH/NODE/KR/BI-01-C
MESH/NODE/KR/BI-02-C
MESH/NODE/KR/BI-03-C
MESH/NODE/KR/BI-04-C
MESH/NODE/KR/BI-05-C
MESH/NODE/KR/BV-01-C
MESH/NODE/KR/BV-02-C
MESH/NODE/KR/BV-03-C
MESH/NODE/MPS/BI-01-C
MESH/NODE/MPS/BV-01-C
MESH/NODE/MPS/BV-02-C
MESH/NODE/MPS/BV-03-C
MESH/NODE/MPS/BV-04-C
MESH/NODE/MPS/BV-05-C
MESH/NODE/MPS/BV-06-C
MESH/NODE/MPS/BV-07-C
MESH/NODE/MPS/BV-08-C
MESH/NODE/MPS/BV-09-C
MESH/NODE/MPS/BV-10-C
MESH/NODE/NET/BI-01-C
MESH/NODE/NET/BI-02-C
MESH/NODE/NET/BI-03-C
MESH/NODE/NET/BI-04-C
MESH/NODE/NET/BV-01-C
MESH/NODE/NET/BV-02-C
MESH/NODE/NET/BV-03-C
MESH/NODE/NET/BV-04-C
MESH/NODE/NET/BV-05-C
MESH/NODE/NET/BV-06-C
MESH/NODE/NET/BV-07-C
MESH/NODE/NET/BV-08-C
MESH/NODE/NET/BV-09-C
MESH/NODE/NET/BV-10-C
MESH/NODE/NET/BV-11-C
MESH/NODE/NET/BV-12-C
MESH/NODE/NET/BV-13-C
MESH/NODE/NET/BV-14-C
MESH/NODE/PBADV/BI-01-C
MESH/NODE/PBADV/BI-02-C
MESH/NODE/PBADV/BI-03-C
MESH/NODE/PBADV/BI-04-C
MESH/NODE/PBADV/BV-01-C
MESH/NODE/PBADV/BV-02-C
MESH/NODE/PBADV/BV-03-C
MESH/NODE/PBADV/BV-04-C
MESH/NODE/PROV/BI-01-C
MESH/NODE/PROV/BI-02-C
MESH/NODE/PROV/BI-03-C
MESH/NODE/PROV/BI-13-C
MESH/NODE/PROV/BV-01-C
MESH/NODE/PROV/BV-07-C
MESH/NODE/PROV/BV-09-C
MESH/NODE/PROV/BV-10-C
MESH/NODE/PROV/BV-11-C
MESH/NODE/PROV/BV-12-C

Reply:
Not all of those are because of the Friend feature. E.g. the last three are the tests that we already have patches for in master & LTS branches for provisioning. I.e. those are for Mesh 1.0.1.

As for MESH/NODE/CFG/HBS/BV-05-C-LT2, like the other "lower tester 2" tests I don't think that's found anywhere in the Test Spec, rather it's just for the second PTS instance, meaning we don't need test evidence from it even though it plays a role in the testing.

Btw, I think you have a typo in the fourth test, it should be LPNPT